src/CheckoutSdk/Marketplace/MarketplacePaymentInstrument.cs

```diff
@@ -0,0 +1,37 @@
+﻿using Checkout.Common;
+using Checkout.Common.Four;
+using Checkout.Instruments;
+
+namespace Checkout.Marketplace
+{
+    public class MarketplacePaymentInstrument
+    {
+        public readonly InstrumentType Type = InstrumentType.BankAccount;
+
+        public string Label { get; set; }
+
+        public AccountType? AccountType { get; set; }
+
+        public string AccountNumber { get; set; }
+
+        public string BankCode { get; set; }
+
+        public string BranchCode { get; set; }
+
+        public string Iban { get; set; }
+
+        public string Bban { get; set; }
+
+        public string SwiftBic { get; set; }
+
+        public Currency Currency { get; set; }
```

Comment:
Please add `?` to enums.

Reply:
Already review by Armando: this enum is in a request object and is mandatory on the documentation.